doc/src/manual/noteworthy-differences.md

```diff
@@ -345,7 +345,13 @@ For users coming to Julia from R, these are some noteworthy differences:
 
 - The typical Julia workflow for prototyping also uses continuous manipulation of the image, implemented with the [Revise.jl](https://github.com/timholy/Revise.jl) package.
 
-- Bignums are supported, but conversion is not automatic; ordinary integers [overflow](@ref faq-integer-arithmetic).
+- For performance, Julia prefers that operations have [type stability](@ref man-type-stability-1). This means that, where Common Lisp numeric operations feel like math, Julia numeric operations feel more like "computer arithmetic". For example:
```

Comment:
I understand the sentiment here, but describing how this kind of thing "feels" seems a bit too subjective. Can you phrase this more objectively?